linera-base/src/identifiers.rs

```diff
@@ -453,6 +497,56 @@ impl StreamId {
     }
 }
 
+/// The result of an events from index.
```

Comment:
If you refer to a function, let's use backticks and the correct function name: `` `events_from_index` ``